src/Arcus.Messaging.Tests.Integration/MessagePump/MessagePumpService.cs

```diff
@@ -0,0 +1,138 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using Arcus.EventGrid;
+using Arcus.EventGrid.Parsers;
+using Arcus.EventGrid.Testing.Infrastructure.Hosts.ServiceBus;
+using Arcus.Messaging.ServiceBus.Core.Extensions;
+using Arcus.Messaging.Tests.Core.Events.v1;
+using Arcus.Messaging.Tests.Core.Logging;
+using Arcus.Messaging.Tests.Core.Messages.v1;
+using Arcus.Messaging.Tests.Integration.Fixture;
+using Bogus;
+using GuardNet;
+using Microsoft.Azure.EventGrid.Models;
+using Microsoft.Azure.ServiceBus;
+using Microsoft.Azure.ServiceBus.Core;
+using Microsoft.Extensions.Configuration;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Arcus.Messaging.Tests.Integration.MessagePump
+{
+    /// <summary>
+    /// Represents a service to interact with the hosted-service.
+    /// </summary>
+    public class MessagePumpService : IAsyncDisposable
```

Comment:
Clearly state that it's for testing?
```suggestion
    public class TestMessagePumpService : IAsyncDisposable
```

Reply:
Yeah, ok.